folge-3/build.gradle

```diff
@@ -69,9 +70,11 @@ task convertToDocx (
         dependsOn: 'generateDocBook',
         type: Exec
 ) {
-  workingDir 'build/asciidoc/docbook'
+  doFirst {
+    file(Paths.get(generateDocBook.outputDir.toString(), "docx")).mkdirs()
+  }
+  workingDir file(Paths.get(generateDocBook.outputDir.toString(), "docbook"))
```

Comment:
ok. using `generateDocBook.outputDir` is cleaner and a `doFirst` is also cleaner but
`new File(generateDocBook.outputDir.canonicalPath+"/docx").mkdirs()`
is shorter and you can omit the include above :-)

Reply:
Yeah, good point.
The `doFirst` was required, though. Didn't work reliable for me in the `simple-meetup` project without.

Reply:
I like the suggested change. But let us agree on `new File(generateDocBook.outputDir.canonicalPath, "docx").mkdirs()` subtle, but much nicer :)

Reply:
that's true and I still don't understand why it does not work reliable. But with your refactoring, I guess I can fix this for docToolchain in general...

Reply:
we should more often do pair coding - I have the feeling that the result could be something big :-)

Reply:
Thanks Ralf :) much appreciated.